apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/CharSequenceToContainAtLeastElementsOfExpectationsTest.kt

```diff
@@ -0,0 +1,149 @@
+        ) = expect.toContain.atLeast(atLeast).butAtMost(butAtMost).elementsOf(listOf(a, *aX))
+
+        private val atLeastButAtMostIgnoringCaseDescr = { what: String, timesAtLeast: String, timesAtMost: String ->
+            "$toContain $ignoringCase $what $atLeast $timesAtLeast $butAtMost $timesAtMost"
+        }
+
+        private fun getAtLeastButAtMostIgnoringCaseElementsOfTriple() =
+            atLeastButAtMostIgnoringCaseDescr to ("$toContain.$ignoringCase.$atLeast.$butAtMost.$elementsOf" to Companion::toContainAtLeastButAtMostIgnoringCaseElementsOf)
+
+        private fun toContainAtLeastButAtMostIgnoringCaseElementsOf(
+            expect: Expect<CharSequence>,
+            atLeast: Int,
+            butAtMost: Int,
+            a: Any,
+            aX: Array<out Any>
+        ) = expect.toContain.ignoringCase.atLeast(atLeast).butAtMost(butAtMost).elementsOf(listOf(a, *aX))
+
+        private fun getNotToContainPair() = toContainNot to ::getErrorMsgNotToContain
+
+        private fun getErrorMsgNotToContain(times: Int) = "use $toContainNot instead of $atLeast($times)"
+
+        private fun getExactlyPair() = exactly to Companion::getErrorMsgExactly
+
+        private fun getErrorMsgExactly(times: Int) =
+            "use $exactly($times) instead of $atLeast($times).$butAtMost($times)"
+
+        internal fun getErrorMsgAtLeastButAtMost(timesAtLeast: Int, timesButAtMost: Int) =
+            "specifying $butAtMost($timesButAtMost) does not make sense if $atLeast($timesAtLeast) was used before"
+
+        private fun <T> iterableLikeToIterableTestFactory(
+            description: String,
+            subject: T,
+            funIterableLike: Expect<T>.(IterableLike) -> Expect<T>,
+            describePrefix: String = "[Atrium] "
+        ) = testFactory {
+            describe(describePrefix + description) {
+                emptyIterableLikeTypes.forEach { (type, input) ->
+                    it("passing an empty $type throws an IllegalArgumentException") {
+                        expect {
+                            expect(subject).funIterableLike(input)
+                        }.toThrow<IllegalArgumentException> {
+                            messageToContain("IterableLike without elements are not allowed")
+                        }
+                    }
+                }
+
+                it("passing a String instead of an IterableLike throws an IllegalArgumentException") {
+                    expect {
+                        expect(subject).funIterableLike("test")
+                    }.toThrow<IllegalArgumentException> {
+                        messageToContain(
+                            "IterableLikeToIterableTransformer accepts arguments of types:",
+                            "Iterable<*>, Sequence<*>, Array<*>, CharArray, ByteArray, ShortArray, IntArray, LongArray, FloatArray, DoubleArray and BooleanArray",
+                            "given:",
+                            "String"
```

Comment:
out of curiosity, why did you change this bit, did it no longer work with `"given: kotlin.String"`?

Reply:
Makes sense, the test did not run on JS so far. Nice to see that we reveal subtle bugs (in this case in the test code, but still)